### ISSUE TITLE
Problems with Roles and users

### ISSUE THREAD

**Issue by scottw12** [vanjarosoftware/Vanjaro.Platform#1949]
**Product & Version**
Vanjaro 1.2.0

**Describe the bug**
Cannot create a new user and give special permissions for editing content as Editor
We do have pages with the Editor being set by permissions to be able to edit.
**To Reproduce**
1. Create a New User
2. Click on Settings -> Roles -> Add icon and add editor role
3.Return to main menu and log out.
4. click on "My Account" link and login in
5. Login is accepted, however I am just returned back to the home page. No indication of being signed in, the login link still shows "My Account"
6. If I go back in as super-user and make the new user a super-user then all works, that user indeed has super-user abilities and works as expected.

Attached is a screen cap that show the above events.  http://somup.com/crlnYu0x6t

**Additional context**
Add any other context about the problem here including the Browser/OS version.

**Comment by ashishpachauriap** [comment]
Is there any error in **Site logs** related to it? Please provide screencast of it.
Also, tell us what is the setting of the editor role.

**Comment by scottw12** [comment]
As posted in the above post:
Attached is a screen cap that show the above events. http://somup.com/crlnYu0x6t
Nothing in the error log.

**Comment by scottw12** [comment]
I am more than willing to allow someone super users access so they can test this. Will need a secure way to send credentials. Or we could even do an online session so we can look at the server itself and do some debugging.

**Comment by Gaganbird** [comment]
Are you able to reproduce the issue on the clean installation. If so please provide the instructions.

**Comment by scottw12** [comment]
No, I cannot reproduce the error on a clean installation of V1.2.0. The issue we are having is in an upgraded version of  V1.2.0

I will go through the steps I used to create a new user.

1 Login in as Super User
2 Goto Site -> Users -> plus sign to add new user. 
3 Create the user. 
4 Click on user and then settings.
5 Add Role of Editor.
6 Sign out as SuperUser.
7 Sign in as the new user. (if I do not enter a valid password I am warned and then have to enter the correct one so that part works)
8 Website returns to home page and the user has not been logged in as shown at the bottom of the screen.
9 Login in a Super User.
10 Goto Site -> Users  
11 Open the settings menu of the user and click on "Log in as User"
12 The website then goes to https://jcesba.org/tabid/0/Default.aspx which lead to a 404 error. 
13 Clear  the /TabId/0/Default.aspx  off the URL and it goes to the home page.
12 User is logged in and all works fine.

Here is a screen cast that follows the above actions.
http://somup.com/crloqJ0KTY
